### PR TITLE
Modernize Synapse instructions

### PR DESCRIPTION
**Pull request by gStart9**

### CODE REVIEW

File: site/source/user-manual/service-guides/matrix/index.rst [hunk 1]
```diff
@@ -0,0 +1,14 @@
+.. _synapse-service:
+
+=======
+Synapse
+=======
+
+Synapse is a self-hosted Matrix server, capable of federation with the wider matrix network (but only .onion for now).
```
Comment (kn0wmad) [inline site/source/user-manual/service-guides/matrix/index.rst:7]:
```suggestion
Synapse is a self-hosted Matrix server, capable of federation with the wider Matrix network (but only .onion for now).
```

File: site/source/user-manual/service-guides/matrix/index.rst [hunk 2]
```diff
@@ -0,0 +1,14 @@
+.. _synapse-service:
```
Comment (kn0wmad) [inline site/source/user-manual/service-guides/matrix/index.rst:1]: As the "Matrix Index," I think this should be labeled as such, then files in this directory can be for services/clients, i.e. Synapse, Conduit, etc

File: site/source/user-manual/service-guides/matrix/synapse.rst [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+Creating a new user
+-------------------
+
+These instructions will guide you through creating your first Matrix user on your Synapse service via the admin interface.
+
+#. Visit the instructions page by clicking `Services > Synapse > Instructions` and following along.
+
+#. As you come to **Step 2: Creating Your First Account**, continue below for instructions augmented with screenshots:
+
+  **Substep 1**: In your Synapse dashboard, click "Launch UI"
```
Comment (kn0wmad) [inline site/source/user-manual/service-guides/matrix/synapse.rst:24]: Maybe check how 'substeps' are done elsewhere.  I think I normally turn them into dashed lists, but it's possible we have used 1a., 1b., etc type listing

**Review by kn0wmad — CHANGES_REQUESTED** [review]
Mainly notes on index page

**Comment by gStart9** [comment]
I actually either want to scrap this change to the docs and just have the instructions updated in the Synapse wrapper:
https://github.com/Start9Labs/synapse-wrapper/pull/58

Or I want the synapse wrapper's instructions to point to the docs where we explain everything there and nothing in that step of the instructions.

@k0gen Can you review my PR?  Do you like it for the Synapse wrapper?

**Comment by kn0wmad** [comment]
Okay, sounds like this can be closed then?

**Pull request closed without merging**